index.html

```diff
@@ -289,6 +289,26 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <div class="contact-form">
+        <section class="contact-form-section">
+          <h1 class="primary-text">
```

Comment:
- [ ] As you can see in the [HTML and CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) we have to use semantic HTML for this project, so in this case, I think it'd be better if this was a `<h2>` tag instead of a  `<h1>`, this helps with accessibility as screen readers and other tools treat `<h1>` as a top-level header